### PR TITLE
Fix tests on master

### PR DESCRIPTION
**Pull request by bkchr**
Because of a bug in the test script, we didn't stopped CI when the main
tests are failed.

### CODE REVIEW

**Review by ordian — APPROVED** [review]
Looks good, nice catch!

File: primitives/src/v1.rs [hunk 1]
```diff
@@ -231,6 +231,11 @@ impl<H: Clone> CommittedCandidateReceipt<H> {
 	pub fn hash(&self) -> CandidateHash where H: Encode {
 		self.to_plain().hash()
 	}
+
+	/// Does this committed candidate receipt corrensponds to the given [`CandidateReceipt`]?
+	pub fn corresponds_to(&self, receipt: &CandidateReceipt<H>) -> bool where H: PartialEq {
```
Comment (ordian) [inline primitives/src/v1.rs:236]: this method is used only for testing in one place, is it worth making it a pub method?

Comment (bkchr) [reply primitives/src/v1.rs:236]: Maybe it is useful somewhere else. I did not scan all the code if we already need something similar somewhere else. 

TLDR: Yes.

**Comment by bkchr** [comment]
bot merge
